pom.xml

```diff
@@ -251,6 +251,12 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- https://mvnrepository.com/artifact/io.confluent/kafka-connect-avro-data -->
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-connect-avro-data</artifactId>
```

Comment:
Should this dependency have `<scope>test</scope>`? I don't see (and didn't expect) any runtime Avro dependencies added for this change?

Reply:
Yes definitely. I was messing up with this to see if I can use at converter itself but then realized it was not necessary :) Made change ea4c7bc522d4ae47844b26fd07edc6003e8b6ee0